src/content/plugins/split-chunks-plugin.md

```diff
@@ -344,3 +345,28 @@ module.exports = {
 ```
 
 W> This might result in a large chunk containing all external packages. It is recommended to only include your core frameworks and utilities and dynamically load the rest of the dependencies.
+
+### Split Chunks: Example 3
+
+ Create a `custom vendors` chunk, which includes only those `node_modules` from the whole application which passes the RegEx Test
+ 
+ __webpack.config.js__
+
+```js
+module.exports = {
+  //...
+  optimization: {
+    splitChunks: {
+      cacheGroups: {
+        vendor: {
+          test: /[\\/]node_modules[\\/](react|react-dom)[\\/]/,
+          name: 'vendor',
+          chunks: 'all',
+        }
+      }
+    }
+  }
+};
+```
+
+ T> This will result into a vendor chunk containing desired libraries provided in RegEx pattern. A better understanding of chunks can be visualized by  [webpack-visualizer](https://chrisbateman.github.io/webpack-visualizer/).
```

Comment:
Not sure we need to promote `webpack-visualizer` here as there might be more appropriate places in docs. Instead, we might wanna say:
This will result in splitting `react` and `react-dom` into a separate chunk.

Reply:
@byzyk  Thanks for the needful input. I'll make recommended changes. I have one concern here sometimes users aren't sure if they custom vendor actually contains only those libraries in RegEx so they can verify it with weback visualizer. Anyways, I'm taking your feedback and making changes in the PR.

Reply:
I have pushed the latest changes.

Reply:
This plugin is already [mentioned in docs](https://webpack.js.org/guides/code-splitting/#bundle-analysis), so if you feel like we need to add a note about that here you can just reference the links I shared, saying something like: 

If you're not sure what packages have been included in a chunk you may refer to Bundle Analysis section(link) for details.

Reply:
Thanks @byzyk  added requested changes.